bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/AbstractHomekitPositionAccessoryImpl.java

```diff
@@ -44,6 +45,7 @@
     protected int closedPosition;
     protected int openPosition;
     private final Map<PositionStateEnum, String> positionStateMapping;
+    private final List<PositionStateEnum> customPositionStateList;
```

Comment:
This isn't used anywhere besides the constructor. Why is this field needed?

Reply:
correct, was copy paste from other classes. but here we would not reduce the list of supported states.
i have removed the field but Im still passing an empty array as parameter to satisfy  @NonNullByDefault of the method. i hope it is fine to create a temporar array instance. or is it better to make the parameter @Nullable?

Reply:
So is the passed in ArrayList instance not even used then?

Reply:
another proposal. i made now the parameter nullable so that i can use "null" instead of array instances, in cases when accessory does not support custom enum lists.
please take a look